site/content/en/guides/producer/setters/_index.md

```diff
@@ -87,6 +87,51 @@ spec:
   replicas: 3 # {"$kpt-set":"replicas"}
 ```
 
+#### Targeting fields using the path
+
+To limit the fields targeted by the setter, use the `--field` flag to provide
+either the name of the field, the full path to the field, or a partial (suffix)
+path to the field. Only fields that match the path will be targeted by the setter.
+
+```sh
+# create a setter named "replicas" and but only target the field name replicas in the spec
+kpt cfg create-setter hello-world/ replicas 3 --field="spec.replicas"
+```
+
+```yaml
+# deployment-foo.yaml
+kind: Deployment
+metadata:
+  name: foo
+  annotations:
+    replicas: 3
+spec:
+  replicas: 3 # {"$kpt-set":"replicas"}
+```
+
+The path is always just the names of the properties on the path to the field,
+regardless whether the field is nested inside a sequence.
+
+```sh
+# create a setter named "mountName" and only target the name of the volume mount
```

Comment:
Probably modify this example to have multiple list values for volumeMounts and describe the current behavior. We might want to have action items to make it the syntax more expressive.

Reply:
Added an additional volumeMount to the example. Added more explanation further up on how targeting works without the `--field` flag.